src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -137,6 +137,17 @@ export const terminalSendSequenceCommand = async (accessor: ServicesAccessor, ar
 	}
 };
 
+export const terminalSendSignalCommand = async (accessor: ServicesAccessor, args: unknown) => {
+	const instance = accessor.get(ITerminalService).activeInstance;
+	if (instance) {
+		const signal = isObject(args) && 'signal' in args ? toOptionalString(args.signal) : undefined;
+		if (!signal) {
+			return;
+		}
+		instance.sendSignal(signal);
+	}
+};
```

Comment:
This should show a quick pick with all possible signals when there is no argument provided.